cqf-fhir-utility/src/main/java/org/opencds/cqf/fhir/utility/visitor/AbstractKnowledgeArtifactVisitor.java

```diff
@@ -32,4 +39,27 @@ protected List<IBaseBackboneElement> findArtifactCommentsToUpdate(
             throw new UnprocessableEntityException("Version not supported");
         }
     }
+
+    protected List<IDomainResource> getComponents(
+            KnowledgeArtifactAdapter adapter, Repository repository, ArrayList<IDomainResource> resourcesToUpdate) {
+        adapter.getOwnedRelatedArtifacts().stream().forEach(c -> {
+            final var preReleaseReference = KnowledgeArtifactAdapter.getRelatedArtifactReference(c);
+            Optional<KnowledgeArtifactAdapter> maybeArtifact =
+                    VisitorHelper.tryGetLatestVersion(preReleaseReference, repository);
+            if (maybeArtifact.isPresent()) {
+                if (resourcesToUpdate.stream()
+                        .noneMatch(rtu ->
+                                rtu.getId().equals(maybeArtifact.get().getId().toString())
+                                        && (rtu.getExtension().stream()
+                                                .anyMatch(ext -> ext.getUrl().equals(isOwnedUrl))))) {
```

Comment:
the `isOwned` extension should be on the `RelatedArtifact`, not in the resource 😅  and since we're using `getOwnedRelatedArtifacts` this check is redundant?